BHoM/LifeCycleAssessment_Toolkit
# Update Boverket datasets

### NOTE: Depends on 



   
### Issues addressed by this PR


Closes #359 




### Test files

To test: 
1. Create a new dataset primitive
2. Right click search and select LifeCycleAssessment > Boverket > select one of the two new datasets and repeat
3. Verify functionality

### Changelog




### Additional comments

Once consensus has been reached with this dataset, I would suggest we remove the legacy dataset, but due to underlying dependencies I have chosen to leave it in place. 

Note I have not validated any numbers and have only pulled their data using their API. I have captured their A1-A3, A4, and A5.1 (as A5 because we don't support subcategories) for both conservative figures and typical figures (only affecting A1-A3 from what I can tell). These values should be tested and validated by the community.
